File: C#.Net/Tests/JDI UIWebTests/JDI UIWebTests/UIObjects/Sections/LoginForm.cs
using Epam.JDI.Core.Attributes;
using Epam.JDI.Core.Interfaces.Base;
using Epam.JDI.Core.Interfaces.Common;
using JDI_UIWebTests.Entities;
using JDI_Web.Selenium.Elements.Composite;
using OpenQA.Selenium.Support.PageObjects;

namespace JDI_UIWebTests.UIObjects.Sections
{
    public class LoginForm : Form<User>
    {
        [FindsBy(How = How.CssSelector, Using = "a>div.profile-photo")]
        private IClickable profile;

        [FindsBy(How = How.CssSelector, Using = "button.btn-login")]
        private IButton loginButton;

        [FindsBy(How = How.Id, Using = "Login")]
        [Name("Login")]
        private ITextField loginField;

        [FindsBy(How = How.Id, Using = "Password")]
        [Name("Password")]
        private ITextField passwordField;

        public new void Submit(User user)
        {
            profile.Click();
            base.Submit(user);
        }
    }
}

File: Java/Tests/jdi-uitest-cucumber-example/src/main/java/org/mytests/uiobjects/example/pages/LoginForm.java
package org.mytests.uiobjects.example.pages;

import com.epam.jdi.uitests.web.selenium.elements.common.Button;
import com.epam.jdi.uitests.web.selenium.elements.common.TextField;
import com.epam.jdi.uitests.web.selenium.elements.composite.Form;
import org.mytests.uiobjects.example.entities.User;
import org.openqa.selenium.support.FindBy;

/**
 * Created by Roman_Iovlev on 10/22/2015.
 */
public class LoginForm extends Form<User> {
    @FindBy(id="Login")
    public TextField name;
    @FindBy(id="Password")
    public TextField password;

    @FindBy(css="[type=submit]")
    public Button enter;
}
